plat/kvm/Linker.uk

```diff
@@ -1,5 +1,6 @@
 ifeq (x86_64,$(CONFIG_UK_ARCH))
 KVM_LDFLAGS-y += -Wl,-m,elf_x86_64
+KVM_LDFLAGS-y += -Wl,--entry _multiboot_entry
```

Comment:
Hi @marcrittinghaus, while I was looking at #603 I spotted this one.
```
  LD      app-helloworld_kvm-x86_64.o
gcc: error: _multiboot_entry: No such file or directory
```
Seems that the syntax is `--entry=_multiboot_entry`.
I'm using GCC 9.4.0.

Reply:
Thanks very much! This does not seems to be a problem with GCC 12. I updated the offending code.